src/org/openstreetmap/josm/io/remotecontrol/handler/RequestHandler.java

```diff
@@ -202,7 +202,7 @@ public final void checkPermission() throws RequestHandlerForbiddenException {
          */
         if (GLOBAL_CONFIRMATION.get()) {
             // Ensure dialog box does not exceed main window size
-            Integer maxWidth = (int) Math.max(200, MainApplication.getMainFrame().getWidth()*0.6);
+            int maxWidth = (int) Math.max(200, MainApplication.getMainFrame().getWidth()*0.6);
```

Comment:
It might not be a bad idea to add spaces around `*` (`getWidth() * 0.6`).

Reply:
Fixed in 5e03cac3.